CHANGELOG.md

```diff
@@ -134,6 +134,7 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
   - webui is NOT affected
   - webui does NOT use the npm server
   - webui does NOT use a user-provided locale string to directly switch moment locale
+- webui: Fix URL rewrite vulnerability in zend-http component [PR #1213] [No known CVE](https://github.com/advisories/GHSA-f6p5-76fp-m248)
```

Comment:
We need a "Breaking changes" entry that CentOS7/RHEL7 don't have webui anymore.

Reply:
Change request applied here https://github.com/bareos/bareos/pull/1213/commits/daa1a2b97bc85b259ca5f3e18d09f6f2a9b2d719